src/Akka.Persistence.SqlServer.Tests/packages.config

```diff
@@ -1,14 +1,28 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Akka" version="1.0.8" targetFramework="net45" />
-  <package id="Akka.Persistence" version="1.0.8.25-beta" targetFramework="net45" />
-  <package id="Akka.Persistence.Sql.Common" version="1.0.8.25-beta" targetFramework="net45" />
-  <package id="Akka.Persistence.TestKit" version="1.0.8.25-beta" targetFramework="net45" />
-  <package id="Akka.TestKit" version="1.0.8" targetFramework="net45" />
-  <package id="Akka.TestKit.Xunit2" version="1.0.8" targetFramework="net45" />
+  <package id="Akka" version="1.0.9.236-beta" targetFramework="net45" />
+  <package id="Akka.Persistence" version="1.0.9.236-beta" targetFramework="net45" />
+  <package id="Akka.Persistence.Query" version="1.0.9.236-beta" targetFramework="net45" />
+  <package id="Akka.Persistence.Query.Sql" version="1.0.9.236-beta" targetFramework="net45" />
+  <package id="Akka.Persistence.Sql.Common" version="1.0.9.236-beta" targetFramework="net45" />
+  <package id="Akka.Persistence.Sql.TestKit" version="1.0.9.236-beta" targetFramework="net45" />
+  <package id="Akka.Persistence.TestKit" version="1.0.9.236-beta" targetFramework="net45" />
+  <package id="Akka.Streams" version="1.0.9.236-beta" targetFramework="net45" />
+  <package id="Akka.Streams.TestKit" version="1.0.9.236-beta" targetFramework="net45" />
+  <package id="Akka.TestKit" version="1.0.9.236-beta" targetFramework="net45" />
+  <package id="Akka.TestKit.Xunit2" version="1.0.9.236-beta" targetFramework="net45" />
   <package id="Google.ProtocolBuffers" version="2.4.1.555" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net45" />
+  <package id="Reactive.Streams" version="1.0.1" targetFramework="net45" />
+  <package id="System.Collections" version="4.0.0" targetFramework="net45" />
   <package id="System.Collections.Immutable" version="1.1.36" targetFramework="net45" />
+  <package id="System.Diagnostics.Debug" version="4.0.0" targetFramework="net45" />
+  <package id="System.Globalization" version="4.0.0" targetFramework="net45" />
+  <package id="System.Linq" version="4.0.0" targetFramework="net45" />
+  <package id="System.Resources.ResourceManager" version="4.0.0" targetFramework="net45" />
+  <package id="System.Runtime" version="4.0.0" targetFramework="net45" />
+  <package id="System.Runtime.Extensions" version="4.0.0" targetFramework="net45" />
+  <package id="System.Threading" version="4.0.0" targetFramework="net45" />
```

Comment:
Which of them are redundant? They are all added by System.Collections.Immutable dependency.